package.json

```diff
@@ -62,9 +62,7 @@
     "prebuild": "rm -rf ./vendor && find . -name 'node_modules' -type d -maxdepth 3 -exec rm -rf {} +",
     "build": "composer install --no-dev && yarn install && yarn run archive",
     "prearchive": "rm -rf $npm_package_name.zip",
-    "archive": "zip -r $npm_package_name.zip . -x **.git/\\* **node_modules/\\*",
-    "postarchive": "yarn run archive:cleanup && rm -rf $npm_package_name && unzip $npm_package_name.zip -d $npm_package_name && rm $npm_package_name.zip && zip -r $npm_package_name.zip $npm_package_name && rm -rf $npm_package_name",
-    "archive:cleanup": "zip -d $npm_package_name.zip .env* .ddev/\\* \\*.idea/\\* .editorconfig tests/\\* .github/\\* .psalm/\\* wordpress_org_assets/\\* \\*.DS_Store \\*README.md \\*.gitattributes \\*.gitignore \\*composer.json \\*composer.lock patchwork.json phpunit.xml.dist .phpunit.result.cache phpcs.xml* psalm*.xml* playwright.config.js wp-cli.yml \\*.babelrc \\*package.json \\*webpack.config.js \\*yarn.lock \\*.travis.yml\\"
+    "archive": "wp dist-archive . ./$npm_package_name.zip --plugin-dirname=$npm_package_name"
```

Comment:
Is `composer-assets-compiler` automatically invoked or does it still go into each module and runs `yarn build` there?

Reply:
assets compiler is running only in the workflow now. I guess we can replace the `install:modules:*` and `build:modules:*` but I can't figure out how to setup it properly, e.g. if I run `ddev composer compile-assets -vvv` again it does not rebuild changes.

```
    Composer Assets Compiler
    starting...
    
    Not processing 'woocommerce/woocommerce-paypal-payments' because already processed.
    Nothing else to process.
    
    done.
```

Reply:
@AlexP11223 You can either pass `--ignore-lock` and/or specify the source directories so it actually picks up file changes:

https://github.com/inpsyde/composer-asset-compiler/blob/v3.x/docs/007-Hash_and_Lock.md#custom-source-files

Reply:
@Biont  hm, weird, still the same with ` --ignore-lock`.

btw what was the purpose of `--env=root`? https://github.com/inpsyde/reusable-workflows/blob/1674a94f9eb5e5572278cef20e305c98440a1df6/.github/workflows/build-plugin-archive.yml#L57C22-L57C32

Reply:
@AlexP11223 use `--ignore-lock=*`